src/test/java/de/samply/share/client/model/centralsearch/UploadStatsTest.java

```diff
@@ -0,0 +1,41 @@
+package de.samply.share.client.model.centralsearch;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.xml.XmlMapper;
+import java.time.LocalDateTime;
+import org.junit.jupiter.api.Test;
+
+public class UploadStatsTest {
+
+  @Test
+  public void serializeAndDeserializeUploadStatsTest() throws JsonProcessingException {
+
+    UploadStats uploadStats = createUploadStats();
+    String stringUploadStats = serialize(uploadStats);
+    UploadStats uploadStats2 = deserialize(stringUploadStats);
+
+    assertTrue(uploadStats.getLastUploadTimestamp().equals(uploadStats2.getLastUploadTimestamp()));
+
+  }
+
+  private UploadStats createUploadStats(){
+    LocalDateTime localDateTime = LocalDateTime.now();
+    return  new UploadStats(localDateTime.toString());
```

Comment:
Please always reformat code before committing. You can configure that in IntelliJ.